ironfish-cli/src/commands/peers/banned.ts

```diff
@@ -32,6 +32,10 @@ export class BannedCommand extends IronfishCommand {
       this.exit(0)
     }
 
+    if (flags.sort !== undefined) {
+      this.log('The `sort` flag is not supported when using the `follow` flag.')
+    }
```

Comment:
I'm not sure if you can modify the sort flag that's returned but if you can you can use the built in feature for this which is `exclusive: ['follow']` on the sort flag.

Reply:
Good call, updated!